odata-csdl/1 Introduction.md

```diff
@@ -10,6 +11,18 @@ using the JavaScript Object Notation (JSON), see [RFC8259](#rfc8259).
 This format is based on the OpenUI5 OData V4 Metadata JSON Format, see
 [OpenUI5](#_OpenUI5), with some extensions and
 modifications made necessary to fully cover OData CSDL Version 4.01.
+:
+
+: varxml
+OData services are described in terms of an [Entity
+Model](#EntityModel). The Common Schema Definition Language (CSDL)
+defines a representation of the entity data model exposed by an OData
```

Comment:
```suggestion
defines a representation of the entity model exposed by an OData
```
I think the „data“ here was not intended, the corresponding section is „Entity Model“